app/scripts/models/account.js

```diff
@@ -108,6 +108,11 @@ define(function (require, exports, module) {
 
       this._boundOnChange = this.onChange.bind(this);
       this.on('change', this._boundOnChange);
+
+      const email = this.get('email');
+      if (email && this._notifier) {
```

Comment:
Adding the test for `this._notifier` spared me the effort of fixing up ~90 unit tests (and you the effort of reviewing them). It shouldn't ever not be set in normal execution.